src/services/AffectService.ts

```diff
@@ -7,7 +7,7 @@ import type { ZodAffectType, ZodAffectCVSSType } from '@/types/';
 import { beforeFetch } from './FlawService';
 
 export async function getAffects(cveOrUuid: string) {
-  const field = isCveValid(cveOrUuid) ? 'cve_id' : 'flaw__uuid';
+  const field = isCveValid(cveOrUuid) ? 'flaw__cve_id' : 'flaw__uuid';
```

Comment:
Wow that went  totally unseen on the review, good catch!

Reply:
Lucky we did! I think #590 can help to solve this potential hazard